### DISCUSSION TITLE
📝 Important note about the `v2023.1.0` release

### DISCUSSION THREAD

**Discussion by rustatian** [roadrunner-server/roadrunner#1425] [General]
Hello, community! 👋🏻

I wanted to clarify the changes in the `v2023.1.0` release. There will not be any breaking changes in terms of API, logs, etc. All plugins (externals) and proto API will be the same. We did these releases for the following reasons:
1. You can clearly see how outdated your current RR binary is. At the moment, it's hard to say how outdated `v2.10` is 😃
2. We will not include any new features between quarterly releases. For example, when RR `v2023.1.0` is released, it will only receive bug/security fixes, making RR more stable.
3. We will open a longer testing window (similar to the Jetbrains EAP releases) for RR. For example, one month before the release, we'll start releasing `v2023.2.0-alpha.1`, and so on.

From the PHP side, since RR is a metapackage, we're removing the whole `require` section from the `composer.json`. So, you're free to use any client you want and not depend on ours. If you want to write a better worker, feel free to do that 😃

To summarize: With these new releases, we want to bring **stability**, not breaking changes ❤️
